src/test/scala/org/mdedetrich/stripe/v1/EventsSpec.scala

```diff
@@ -28,26 +33,34 @@ class EventsSpec extends WordSpec with Matchers {
       s"parse accounts.update from $filename" in {
         val in = this.getClass.getResourceAsStream(s"/events/$filename")
         in should not be null
-        val json                = Json.parse(in)
-        val JsSuccess(event, _) = json.validate[Event]
+
+        val string = scala.io.Source.fromInputStream(in).mkString
```

Comment:
I think in future we should be using utility methods for this. This shouldn't block the merge though.